package.json

```diff
@@ -1,17 +1,26 @@
 {
   "name": "react-native-swipedeck",
-  "version": "0.3.0-beta.14",
+  "version": "0.3.0-beta.26",
   "description": "UI for quickly making yes/no decisions on a set of cards",
-  "main": "SwipeCards.js",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "./dist/index.d.ts",
```

Comment:
Consuming apps written in TypeScript will be able to use this typedef to provide automatic type checking at the call site